org.mixedrealitytoolkit.uxcore/Slider/Visuals/SliderVisuals.cs

```diff
@@ -86,7 +86,8 @@ private void Update()
             trackArea.transform.localRotation = Quaternion.Euler(new Vector3(0, 0, angle));
             handle.transform.localRotation = Quaternion.Euler(new Vector3(0, 0, angle));
 
-            trackArea.localScale = new Vector3(SliderState.SliderTrackDirection.magnitude, trackArea.localScale.y, trackArea.localScale.z);
+            Vector3 localSliderTrackVector = trackArea.parent.InverseTransformVector(SliderState.SliderTrackDirection); //does trackArea always have a parent?
```

Comment:
> //does trackArea always have a parent?

Good question! It's probably worth null-checking and only taking this path if it's non-null. Otherwise, follow the existing logic

Reply:
I've added the null check

Reply:
Seems good to me, thanks!